http/src/test/resources/reference.conf

```diff
@@ -308,6 +308,16 @@ azure {
            max-attempts = 120
            interval = 1 seconds
        }
+       runtime-defaults {
+        acr-credential {
+           username = "username"
```

Comment:
Are these actual credentials, or are they templated in via firecloud-develop?

Reply:
they're templated in firecloud-develop, which is why they only exist in [http/src/test/resources/reference.conf](https://github.com/DataBiosphere/leonardo/pull/2650/files/6b234b16d61f7d933d2d8f3bf94786068db4180e#diff-d7d5667e9c02913aa71ba812a20d79cf8999b9bf04584264d67e386adfb992b6) but not the main app reference.conf